### PR TITLE
Changes to home card as suggested

### PR DESCRIPTION
**Pull request by TonyGreenwood**
See changes. Requested by UX.

### CODE REVIEW

**Comment by MichelML** [comment]
@TonyGreenwood for style changes a before/after (pictures) preview is nice to have in the PR

File: scss/components/home-card.scss [hunk 1]
```diff
@@ -17,6 +17,9 @@
 
   display: flex;
   margin-bottom: $home-card-margin;
+  padding-bottom: 1rem;
```
Comment (GermainBergeron) [inline scss/components/home-card.scss]: I'm fine with either, but I think `padding-bottom: 2 * $spacing;` would be the best (same thing for padding top)

File: scss/components/home-card.scss [hunk 2]
```diff
@@ -17,6 +17,9 @@
...
+  padding-top: 1rem;
+  border: 1px solid rgba(181, 196, 207, 0.3);
```
Comment (GermainBergeron) [inline scss/components/home-card.scss]: border: 1px solid rgba($grey-5, $transparency-4);

File: scss/variables.scss [hunk 3]
```diff
@@ -528,7 +528,7 @@ $flippable-transition-duration: 0.8s;
 $flippable-shadow: 0 2px 8px 0 rgba(0, 0, 0, 0.3);
 
 // Material Card
-$material-card-shadow: 0 2px 4px 0 rgba(0,0,0,0.2);
+$material-card-shadow: 0 2px 20px 0 rgba(0,0,0,0.2);
 $material-card-shadow-hover: 0 6px 12px 0 rgba(0,0,0,0.4);
 $material-card-shadow-active: 0 8px 12px 0 rgba(0,0,0,0.5);
```
Comment (GermainBergeron) [inline scss/variables.scss]: I wonder if it will look weird in the hover and active state 🤔

Comment (TonyGreenwood) [reply scss/variables.scss]: Sending you a .mov in DM

**Comment by TonyGreenwood** [comment]
Les changements du deuxieme commit ont été faits et approuvés en collab avec CEBelanger.

Ajout aux material-cards des changements, tenant compte de hover state et active state

**Comment by MichelML** [comment]
good job @TonyGreenwood

**Pull request merged**